sdk/keyvault/azsecrets/models.go

```diff
@@ -88,9 +88,13 @@ type Attributes struct {
 }
 
 func (s Attributes) toGenerated() *internal.SecretAttributes {
+	var recLevel *internal.DeletionRecoveryLevel
+	if s.RecoveryLevel != nil {
```

Comment:
Is this another case where the `toGenerated()` can be simplified?

Reply:
This is the same as the other PR, fixed after merging conflicts